satellite/modules/vm_manage.py

```diff
@@ -85,6 +138,20 @@ def get_memory(compute_ip, vm_name, vm_ip):
 
 
 def vm_details(compute_name, compute_ip, vm_id):
+    """Find details of virtual machine
+
+    In this function virtual machine details like ID, name, state(Running or shut),
+    virtual CPUS, Total memory allocated, Free memory, vm's IP address, vm's MAC address,
+    Cumpute name( on which it is provisioned).
```

Comment:
what is **Cumpute**?

Reply:
make it Compute Name (on which it is provisioned)